packages/front-end/components/Settings/AdminSetPasswordModal.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import Field from "../Forms/Field";
+import Modal from "../Modal";
+import { useForm } from "react-hook-form";
+import { MemberRole } from "back-end/types/organization";
+import { useAuth } from "../../services/auth";
+
+type Member = { id: string; name: string; email: string; role: MemberRole };
+
+type Props = {
+  member: Member;
+  close: () => void;
+};
+
+export default function AdminSetPasswordModal({ member, close }: Props) {
+  const [passwordResetSuccess, setPasswordResetSuccess] = useState(false);
```

Comment:
I think this is a useful functionality that should be part of the Modal component directly.  If you add a new `successMessage` prop to the Modal, then it can take care of swapping out the modal contents, hiding the submit button, and changing the close CTA automatically.  I can see us using this many different places.

Reply:
Just finished the first pass at moving this functionality to the `Modal` component. I need to do some regression testing to make sure I didn't mess up any existing functionality.

Reply:
Alrighty - did some regression testing and didn't bump into anything. It looks like there are a few areas in the app that use the flow above (having a success message, removing the submit button, and updating the close CTA). Most notably, the 'PasswordReset' modal via the top-nav. So, we'll probably want to update that at some point to consume this new `successMessage` prop, but not sure if that should be done in this PR.

Reply:
@jdorn Let me know if the solution I came up with is what you were thinking.